src/components/Popover/types.ts

```diff
@@ -39,4 +39,4 @@ type PopoverProps = BaseModalProps & {
 
 type PopoverWithWindowDimensionsProps = PopoverProps & WindowDimensionsProps;
 
-export type {PopoverProps, PopoverWithWindowDimensionsProps};
+export type {PopoverProps, PopoverWithWindowDimensionsProps, AnchorAlignment, PopoverDimensions};
```

Comment:
Why do we need to export those types? `AnchorAlignment` and `PopoverDimensions` aren't used in any other file it seems